package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "karma start ./karma.conf.js",
     "clean": "del ./dist",
-    "lint": "eslint ./examples ./src ./tests",
+    "lint": "eslint . --fix",
```

Comment:
this will cause circleci to pass even if the code is wrong.
can you maybe add another script that uses `--fix` and leave this one as just a test?

Reply:
`--fix` is mainly for prettier.

If there are some lint errors(unfixable), lint (and CI) fails as usual.

Reply:
I think there is a problem if the result of `npm run lint` is different from what's committed though. I tested it earlier and the return code is 0 even if something changed.

Reply:
OK, I finally put both `lint` and `lint:ci`.

I don't know why but `npm run lint` in CI env fails every time.
(because of platform? prettier version? cache?)
`lint:ci` has `--fix` flag, then able to ignore diffs by prettier.